specification/azsadmin/resource-manager/commerce/Microsoft.Commerce.Admin/preview/2015-06-01-preview/Commerce.json

```diff
@@ -105,7 +105,7 @@
         "tags": [
           "Commerce"
         ],
-        "operationId": "UpdateEncryption",
+        "operationId": "Commerce_UpdateEncryption",
```

Comment:
pls remove this commit, there is a separate PR for this commit https://github.com/Azure/azure-rest-api-specs/pull/13283

Reply:
reverted